### ISSUE TITLE
Training code on LRW/LRW1000?

### ISSUE THREAD

**Issue by Fengdalu** [mpc001/Lipreading_using_Temporal_Convolutional_Networks#6]
Hi, this project seems to contain only test code. Is training codes available？

**Comment by brais-martinez** [comment]
@Fengdalu yes, this is just test code.

@njan-creative In principle we don't have plans to release the training code for now. In any case, I believe that any standard training loop would do, as long as you're careful in setting the training parameters as in the paper.

I guess a question for you two would be, why are you interested in the training code? how would its release help you?

**Comment by Fengdalu** [comment]
@brais-martinez  In fact, I am curious about the cosine lr scheduler, what is the T_max setting in this project?

**Comment by njan-creative** [comment]
@brais-martinez Thanks for the response..

I am new to lip reading.. 
I was looking for a strong baseline to start my work (academic)..
If you are sharing, it would be helpful..

**Comment by brais-martinez** [comment]
I see... I am not sure there is much left to do though, if you're going to do research on lipreading, it would be a 101 lesson to implement the training. I will let you know if in the end we release the training code too, but I am not too sure we'll do.

**Comment by njan-creative** [comment]
Thanks for the reply..

**Comment by farah2410** [comment]
> @brais-martinez Thanks for the response..
> 
> I am new to lip reading..
> I was looking for a strong baseline to start my work (academic)..
> If you are sharing, it would be helpful..
have you reach to the train code?

**Comment by farah2410** [comment]
> @brais-martinez Thanks for the response..
> 
> I am new to lip reading..
> I was looking for a strong baseline to start my work (academic)..
> If you are sharing, it would be helpful..
have you reach to the train code??

**Comment by farah2410** [comment]
> @brais-martinez Thanks for the response..
> 
> I am new to lip reading..
> I was looking for a strong baseline to start my work (academic)..
> If you are sharing, it would be helpful..
have you reach to the train code?

**Comment by mpc001** [comment]
#12
